README.md

```diff
@@ -30,6 +35,9 @@ nix-channel --update darwin
 darwin-rebuild changelog
 ```
 
+> NOTE: If you are using nix as a daemon service the channel for that will be owned by root.
+> Use `sudo -i nix-channel --update darwin` instead.
```

Comment:
```suggestion
> NOTE: If you are using a multi-user install of Nix, which is now the default, the Nix daemon will use root's channels.
>These need to be updated as root and because of the configuration of `sudo` on macOS we need to pass the `-i` flag.
> Use `sudo -i nix-channel --update darwin` instead.
```